CHANGELOG.md

```diff
@@ -1,88 +1,89 @@
-
-## [1.0.1] - 2016-10-07
-
-### Fixed
-- Removed specified stemcell version from tile.yml so that most recent stemcell is
-used by default.
-
-## [2.0.0] - 2016-10-10
-
-### Fixed
-- fixed CloudSQL docs link in README
-- updated credentials type returned by bind call to be a map[string]string instead
-of a string.
-
-## [2.0.1] - 2016-10-28
-
-- CloudSQL will generate a username/password on bind if one is not provided.
-
-### Fixed
-- CloudSQL custom plans are now optional
-- fixed username and password env var names in docs
-- fixed CloudSQL custom plan names and/or features not updating
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unresolved]
+
+### Added
+- New sub-command `plan-info` to dump plan information to the console.
+- New sub-command `client` to execute documentation examples and interact with the broker.
+- New sub-command `help` which outputs help documentation.
+- New sub-command `config` which can convert between configuration file formats.
+- New sub-command `generate` to generate use, tile forms, and configuration documentation.
+- New sub-command `serve` to run the service broker server.
+- New sub-command `show` to dump database state.
+- The ability to configure the system with YAML, TOML, properties, or JSON configuration files via the `--config` flag in conjunction with using environment variables.
+- The ability to customize the database name in the Tile.
+- The ability to turn on/off services via environment variable.
+- Default plans for Spanner, BigTable, and CloudSQL.
+
+### Deprecated
+- Running the service by executing the main executable. Use the `serve` sub-command instead.
+
+### Changed  
+- **Breaking** plan ids are now required and will not be generated if not supplied.
+- **Breaking** changed custom plan id field name from `guid` to `id`.
+- **Breaking** modified `"features"` plan config field name to `"service_properties"`.
+- **Breaking** modified structure of all catalog-related environment variables - `plans` is now a sub-field of the Service object,
+and Service objects are defined individually by setting env variables like `GOOGLE_<SERVICE_NAME>`
+- You no longer have to specify service information in the manifest.
+- **Breaking** The GCS plan `reduced_availability` was changed to `reduced-availability` to be compliant with the spec and work with Kubernetes.
+- Tables are created only if they do not exist on migration, fixing [#194](https://github.com/GoogleCloudPlatform/gcp-service-broker/issues/194).
+- The broker now adheres to OSB version 2.13.
 
-## [2.0.2] - 2016-11-16
+## [3.6.0] - 2017-01-03
 
-### Fixed
-- fixed bug where CloudSQL was returning 400s for all 2nd gen instance provision requests
+- changed default authorized networks for cloudsql instances from `0.0.0.0/0` to none
```

Comment:
Nit, the use of capitalization and periods in this section deviates from the rest of the doc.